codegen/library.xml

```diff
@@ -15,48 +15,40 @@
+    <password>honeybadger</password>
+  </jdbc>
+  <generator>
+    <!-- The default code generator. You can override this one, to generate your own code style
              Defaults to org.jooq.util.DefaultGenerator -->
-        <name>org.jooq.util.DefaultGenerator</name>
-
-        <database>
-            <!-- The database type. The format here is:
+    <name>org.jooq.util.DefaultGenerator</name>
+    <database>
+      <!-- The database type. The format here is:
                  org.util.[database].[database]Database -->
-            <name>org.jooq.util.mysql.MySQLDatabase</name>
-
-            <!-- The database schema (or in the absence of schema support, in your RDBMS this
+      <name>org.jooq.util.mysql.MySQLDatabase</name>
+      <!-- The database schema (or in the absence of schema support, in your RDBMS this
                  can be the owner, user, database name) to be generated -->
-            <inputSchema>killbill</inputSchema>
-
-            <!-- All elements that are generated from your schema
+      <inputSchema>killbill</inputSchema>
+      <!-- All elements that are generated from your schema
                  (A Java regular expression. Use the pipe to separate several expressions)
                  Watch out for case-sensitivity. Depending on your database, this might be important! -->
-            <includes>authorize_net_payment_methods|authorize_net_customer_profiles|authorize_net_requests|authorize_net_transactions</includes>
-
-            <!--Don't include the schema name in the generated classes or the sql they generate -->
-            <outputSchemaToDefault>true</outputSchemaToDefault>
-
-            <!-- All elements that are excluded from your schema
+      <includes>authorize_net_payment_methods|authorize_net_customer_profiles|authorize_net_requests|authorize_net_transactions</includes>
+      <!--Don't include the schema name in the generated classes or the sql they generate -->
+      <outputSchemaToDefault>true</outputSchemaToDefault>
+      <!-- All elements that are excluded from your schema
                  (A Java regular expression. Use the pipe to separate several expressions).
                  Excludes match before includes -->
-            <excludes></excludes>
-        </database>
-
-        <target>
-            <!-- The destination package of your generated classes (within the destination directory) -->
-            <packageName>com.womply.billing.killbill.plugins.jooq</packageName>
-
-            <!-- The destination directory of your generated classes -->
-            <directory>../src/generated/java</directory>
-        </target>
-    </generator>
+      <excludes/>
+    </database>
+    <target>
+      <!-- The destination package of your generated classes (within the destination directory) -->
+      <packageName>com.womply.billing.killbill.plugins.jooq</packageName>
+      <!-- The destination directory of your generated classes -->
+      <directory>src/generated/java</directory>
```

Comment:
why did the path change?

Reply:
Jooq codegen for this repo is utterly broken. It needs to be run manually and then the code that is generated gets committed as part of the repo(!). The reason I switched the path here is I feel the manual command that needs to be run should be run inside the repo's directory, not immediately outside it.

BTW, before I forget, that manual command line is something like...

```
java -classpath codegen:lib/jooq-3.6.2.jar:lib/jooq-meta-3.6.2.jar:lib/jooq-codegen-3.6.2.jar:lib/mysql-connector-java-5.1.36.jar:. org.jooq.util.GenerationTool library.xml
```

Reply:
Have we tried to build killbill-resource with these changes I'm not opposed at all to cleaning up the the codegen I just want to be sure there is nothing in how it's built for our use that will break by changing these configurations.

Reply:
Addressed in the latest force pushed commit.